tests/output/ConsoleInteractionTest.py

```diff
@@ -755,7 +755,7 @@ def test_print_results_multiple_ranges(self):
             print_results(
                 self.log_printer,
                 Section(''),
-                [Result('ClangCloneDetectionBear',
+                [Result('CloneDetectionBear',
```

Comment:
dont use semi-real values.  that is confusing.

make it obviously fake, so the reader knows it is fake.

Reply:
okk